agent/consul/usagemetrics/usagemetrics_oss.go

```diff
@@ -25,6 +30,11 @@ func (u *UsageMetricsReporter) emitPeeringUsage(peeringUsage state.PeeringUsage)
 		float32(peeringUsage.Peerings),
 		u.metricLabels,
 	)
+	metrics.SetGaugeWithLabels(
```

Comment:
nice! since these are all additive changes, they merge cleanly OSS-->ENT but it would be nice if we had ENT tests mirroring this. If that adds any value of course.

Reply:
Will add in ENT project

Reply:
Alternatively, if these are additive and the ENT tests aren't any different, can we just add them here?

Reply:
It looks like the ENT tests will need updates, similar to the test additions here.